### PR TITLE
BITMAKER-2100 estela-web: Implement "In Projects/Scheduled Jobs"

### PR DESCRIPTION
**Pull request by raymond1242**
# Description

estela-web: Implement "In Projects/Scheduled Jobs" mockup

# Issue

https://tasks.bitmaker.dev/issues/2100

## Checklist before requesting a review

- [x] I have performed a self-review of my code.
- [x] My code follows the style guidelines of this project.
- [x] I have made corresponding changes to the [documentation](https://github.com/bitmakerla/estela/tree/main/docs).
- [x] New and existing tests pass locally with my changes.
- [x] If this change is a core feature, I have added thorough tests.
- [x] If this change affects or depends on the behavior of other _estela_ repositories, I have created pull requests with the relevant changes in the affected repositories. Please, refer to our [official documentation](https://estela.bitmaker.la/docs/).
- [x] I understand that my pull request may be closed if it becomes obvious or I did not perform all of the steps above.

### CODE REVIEW

**Comment by raymond1242** [comment]
@Adriana618 Could you review this PR pls
https://tasks.bitmaker.dev/issues/2100

**Review by Adriana618 — CHANGES_REQUESTED** [review]
What is the difference between this PR and this:https://github.com/bitmakerla/estela/pull/101?

**Comment by raymond1242** [comment]
@Adriana618 The PR [#101](https://github.com/bitmakerla/estela/pull/101) add role field on ProjectList component. And this PR is about styling ProjectCronjobList component, algo Cronjob Create component. Are very different

**Comment by raymond1242** [comment]
Rebase with main branch

File: estela-web/src/components/ProjectListPage/index.tsx [hunk 1]
```diff
@@ -60,8 +67,15 @@ export class ProjectListPage extends Component<unknown, ProjectsPageState> {
...
-            this.setState({ projects: [...projects], count: data.count, current: data.current, loaded: true });
+            const projectData: ProjectList[] = data.data.map((project: Project, id: number) => {
+                return {
+                    name: project.name,
+                    pid: project.pid,
+                    role: "Admin",
+                    key: id,
+                };
+            });
+            this.setState({ projects: [...projectData], count: data.count, current: data.current, loaded: true });
```
Comment (Adriana618) [inline estela-web/src/components/ProjectListPage/index.tsx:78]: Probably you have conflict with this PR https://github.com/bitmakerla/estela/pull/101 if you merge both.

**Review by Adriana618 — APPROVED** [review]
Looks good.

**Pull request merged**